bwapi/BWAPI/BWAPI.vcxproj

```diff
@@ -352,7 +354,17 @@ popd</Command>
     <ClCompile Include="Source\Thread.cpp" />
     <ClCompile Include="Source\WMode.cpp" />
   </ItemGroup>
+  <ItemGroup>

```

Comment:
Please remove everything you added here and below. Nuget automatically reads `packages.config` and doesn't need this.